### DISCUSSION TITLE
Add a option to select different refresh rate along with screen resolution.

### DISCUSSION THREAD

**Discussion by tasnimtamim** [ppy/osu#22479] [Ideas]
My monitor supports 1600x900@60hz by default. However you can add custom resolution from NVIDIA Control Panel (even works on AMD) to display at 1600x900@89hz. And it works perfectly stable for over 1.5 years now. However I cannot get osu!Lazer to use this custom refresh rate on full screen. It defaults to the usual 1600x900@60hz. Only works when I use borderless, which as much as I prefer it over fullscreen exclusive, introduces input latency.
I want to see the option to select refresh rate directly from osu! settings menu. Games like Rainbow Six Siege can do it, which separates resolution and refresh rate selection, and selecting it works flawlessly. 
[Someone else had same idea as me](https://github.com/ppy/osu/issues/3879), which was put into backlog but then removed for reasons I can't tell. It's a very important feature for people using custom resolutions to add better refresh rate.

**Comment by bdach** [comment]
We use SDL to [get supported window modes](https://github.com/ppy/osu-framework/blob/4346389de2087ed8b718b32506f565d22e48ebdc/osu.Framework/Platform/SDL2DesktopWindow_Windowing.cs#L625-L632). Before checking anything else, it should be checked if SDL even sees modes with those driver-provided FPS values.

**Reply by tasnimtamim** [reply]
Not sure about that or how to check it. Windows Display Adapter settings at least recognizes it and separates 60Hz and my custom refresh rate resolution. Most fullscreen game does use the standard 60Hz unless you have the option to specify using it (like R6S) or just use borderless.
![image](https://user-images.githubusercontent.com/52713990/216132562-2352413a-9999-4980-88be-e23814bc2709.png)
